.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -36,7 +36,8 @@ jobs:
       env:
         DEBIAN_FRONTEND: noninteractive
         IGNORES: "append-* debug_notice transparent_decompression-*
-          transparent_decompress_chunk-* plan_skip_scan-12 pg_dump"
+          transparent_decompress_chunk-* plan_skip_scan-12 pg_dump 
+          dist_move_chunk dist_param"
```

Comment:
I think this PR is fine as-is but I was wondering if instead of this - can't we add something to the build system?
since cmake is generating a make(or ninja) that's more complicated than in normal cases; one option could be to put it inside `pg_regress.sh`... do you know about other options which could be considered?

Reply:
That would make it difficult to manage the scheduled runs.